src/saving/InProgressLoad.cs

```diff
@@ -13,6 +13,12 @@
 /// </remarks>
 public class InProgressLoad
 {
+    /// <summary>
+    ///   Used to stop quick load starting while a load is in progress already
+    ///   True when a save is currently being loaded
+    /// </summary>
+    public static bool IsLoading{ get; private set; }
```

Comment:
Put the explanation for what it is first, then put the one specific use case description.